apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/AbstractR4ResourceProvider.java

```diff
@@ -249,6 +250,9 @@ public Bundle findByPatient(
                 getResourceTypes(), mbiString, isHashed, excludeSamhsa, lastUpdated, serviceDate);
       }
 
+      if (isHashed) {
+        TransformerUtils.logMbiHashToMdc(mbiString);
+      }
```

Comment:
Should we move this up to the top, just after the `isHashed` computation, incase any logging logic is mistakenly added before it?

Reply:
Sounds good.